tensorvis/tensorvisbase/CMakeLists.txt

```diff
@@ -110,23 +114,23 @@ ivw_group("Shader Files" ${SHADER_FILES})
 #--------------------------------------------------------------------
 # Add Unittests
 set(TEST_FILES
-	${CMAKE_CURRENT_SOURCE_DIR}/tests/unittests/tensorvisbase-unittest-main.cpp
-	${CMAKE_CURRENT_SOURCE_DIR}/tests/unittests/arithmic-operations.cpp
-	${CMAKE_CURRENT_SOURCE_DIR}/tests/unittests/de_normalization.cpp
-	${CMAKE_CURRENT_SOURCE_DIR}/tests/unittests/distance-measures.cpp
-	${CMAKE_CURRENT_SOURCE_DIR}/tests/unittests/set-operations.cpp
-	${CMAKE_CURRENT_SOURCE_DIR}/tests/unittests/to-string.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/tests/unittests/tensorvisbase-unittest-main.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/tests/unittests/arithmic-operations.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/tests/unittests/de_normalization.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/tests/unittests/distance-measures.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/tests/unittests/set-operations.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/tests/unittests/to-string.cpp
 )
 ivw_add_unittest(${TEST_FILES})
 
 if (NOT OPENMP_FOUND)
-	find_package(OpenMP QUIET)
-	if (OPENMP_FOUND)
-		MESSAGE(STATUS "OpenMP flag set in TensorVisBase module.")
-		set (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${OpenMP_C_FLAGS}")
-		set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${OpenMP_CXX_FLAGS}")
-		set (CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} ${OpenMP_EXE_LINKER_FLAGS}")
-	endif()
+    find_package(OpenMP QUIET)
```

Comment:
There is a global setting for openmp. Why not use that one?

Reply:
Usually, when I'm not using things that already exist it's because I'm unaware of their existence :)